src/pages/graphql/training/workshops/index.js

```diff
@@ -1,224 +1,114 @@
 import React from 'react'
-import { navigate } from 'gatsby'
 
-import { BOOTCAMP } from 'src/../images/imageNames.js'
 import Layout from 'src/components/layout'
+import { LinkButton } from 'src/components/buttons'
+import Link from 'src/components/navigation/Link'
 import { TopSection } from 'src/components/layout/Section'
 import Grid, { Col, Row } from 'src/components/layout/Grid'
-import { H2, H3, H4, P } from 'src/components/text'
-import { Link } from 'src/components/navigation'
-import { Card } from 'src/components/elements'
-import { Button } from 'src/components/buttons'
+import { H2, P, H4, H5 } from 'src/components/text'
+import { UpcomingTrainingSection, TrainingCard } from 'src/components/training'
 import Header from 'src/components/layout/Header'
-import {
-  UpcomingTrainingSection,
-  selectUpcomingTrainings,
-  selectNthTraining,
-} from 'src/components/training'
-import { REACT_BOOTCAMP } from 'src/config/data'
-import { InputField, Form, CheckboxField } from 'src/components/form'
-import {
-  composeValidators,
-  required,
-  mustBeEmail,
-} from 'src/components/form/validations'
-import { aliasComponent } from 'src/components/utils/aliasComponent'
-import { EmailInput } from 'src/components/form/Contact'
-import { triggerSessionSubscribe } from 'src/api'
+import { Card, Newsletter } from 'src/components/elements'
 import { Breadcrumb } from 'src/components/navigation'
+import { GRAPHQL_PINK } from '../../../../config/styles'
 
-const NameInput = aliasComponent(InputField)
+const PROVISIONAL_WORKSHOP_PRICE = '£360 Inc VAT'
```

Comment:
we can get price via API now at least for active ones (with instances) and default the others to 360 :-)